src/app/pages/quickMaps/pages/baselineDetails/householdSupply/householdSupply.component.html

```diff
@@ -25,14 +25,17 @@
     </mat-tab>
 
     <mat-tab label="Download">
-      <app-download [chartDownloadPNG]="chartPNG" [chartDownloadPDF]="chartPDF"></app-download>
+      <app-download [chartDownloadPNG]="chartPNG" [chartDownloadPDF]="chartPDF" [csvObject]="csvDownloadData">
+      </app-download>
     </mat-tab>
 
-    <mat-tab label="About {{title}}">
+    <mat-tab label=" About {{title}}">
```

Comment:
I know spaces are ignored usually, but was this vscode formatter adding this pre-space?